assets/src/edit-story/elements/media/edit.js

```diff
@@ -134,8 +135,35 @@ function MediaEdit({ element, box }) {
 
   const isImage = 'image' === type;
   const isVideo = 'video' === type;
+
+  const wheelHandler = useCallback(
+    (evt) => {
+      let newScale = scale || 100;
+      if (event.deltaY > 0) {
```

Comment:
`event` or `evt`?

Reply:
TIL `window.event` stores the last event handled so both work! wow.